lua/packer.lua

```diff
@@ -38,6 +40,7 @@ local config_defaults = {
       get_bodies = 'log --color=never --pretty=format:"===COMMIT_START===%h%n%s===BODY_START===%b" --no-show-signature HEAD@{1}...HEAD',
       submodules = 'submodule update --init --recursive --progress',
       revert = 'reset --hard HEAD@{1}',
+      revert_to = 'reset --hard %s --',
```

Comment:
Question: will this work on new installation ? Say I newly install the plugin then I want to restore a snapshot that I have from older installation. It'll be useful if you're setting up config in a new device or temporarily on a server . I'm asking because packer makes shallow clones that means the commit reference might not exist in newly created clone .

Reply:
What is we just run `git fetch --depth 999999 --progress` before rolling back?